types/setting.go

```diff
@@ -956,6 +962,16 @@ var (
 		Default:  "1",
 	}
 
+	SettingDefinitionRecurringJobMaxRetain = SettingDefinition{
+		DisplayName: "Recurring Job Maximum Retain Number",
```

Comment:
Change the display name to `Maximum Retention Number for Recurring Job`.

Reply:
Modified. Thanks